include/chopper/pack/hierarchical_binning.hpp

```diff
@@ -27,6 +28,9 @@ class hierarchical_binning
     //!\brief The total query cost of all k-mers.
     double total_query_cost{};
 
+    //!\brief The model of the current level IBF for statistics generation
+    hibf_model::ibf * const ibf_model{nullptr};
```

Comment:
If this data object is purely to generate some statistics for us, I would not call it `model` as it suggests that it does more. It should be clear that this has nothing to do with the algorithm.

We should rename it to something like `ibf_statistics`, `ibf_stats` or `ibf_debug_info`?